Updates localhost doc to add a deployable example

nakomis opened the pull request:
Also bumps some of the dependencies, which was required
to build the docs

tbouron commented on README.md:
````diff
@@ -36,7 +36,7 @@ Once the `node` and `npm` are installed, run this command to install all the req
 
 ```bash
-npm install
+npm install; npm install natives
````
There is a huge warning about this package: https://www.npmjs.com/package/natives

Better to go with the recommandation and install `graceful-fs` instead

tbouron commented on package.json:
```diff
@@ -3,7 +3,9 @@
   "main": "index.js",
-  "dependencies": {},
+  "dependencies": {
+    "natives": "^1.1.6"
```
Should go into the `devDependencies`

nakomis commented:
@tbouron As discussed offline, `natives` is (unfortunately) still required as `graceful-fs` doesn't work for us. I have, however, moved it into `devDependencies` and removed the `npm install natives` from the README.md
